api/python/provisioner/inputs.py

```diff
@@ -1112,7 +1112,27 @@ def pillar_value(self):
 
             return res
         elif self.is_remote():
-            return self.source
+            return {
+                f'os': {
+                    'source': (f'{self.source}/os'),
```

Comment:
we assume that remote repo is actually a remotely mounted ISO so directory names should be the same as for ISO. So, please use these constants instead of hard-coed values (e.g. here it should be `config.OS_ISO_DIR`)

Reply:
Done